content/en/guides/models/track/runs/_index.md

```diff
@@ -348,26 +348,30 @@ https://wandb.ai/<team-name>/<project-name>/runs/<run-id>
 Where values enclosed in angle brackets (`< >`) are placeholders for the actual values of the team name, project name, and run ID.
 
 ### Customize how runs are displayed
-This section shows how to customize how runs are displayed in your project's **Workspace** and **Runs** tab, which share the same display configuration.
+This section shows how to customize how runs are displayed in your project's Workspace and Runs table.
```

Comment:
I had to read this a couple times to understand it because of the combination of the casing and the lack of bold. I wonder if it would be clearer if Runs table were before Workspace, and if workspace were not capitalized.